.travis.yml

```diff
@@ -1,28 +1,36 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
```

Comment:
Where is the environment variable `OS` used? I could not find the `OS` in a list of `Dockerfile`.
https://hub.docker.com/_/fedora/
https://hub.docker.com/r/jsmigel/centos-epel/